WordPress/gutenberg
# Block Library: Unify block title with a strict title case

## What?
This PR updates the block title according to the following title rules

- Capitalize the first letter of major words (nouns, verbs, adjectives, adverbs, etc.)
- Other small words (articles, prepositions, conjunctions, etc.) should remain lower case

## Why?
I think it makes sense to apply the same rules to the same features, values, or properties.

The rules regarding capitalization are clearly stated in [Copy Guidelines](https://github.com/WordPress/gutenberg/blob/48b30f4b7da156293b4ffd568db4f879eacde852/docs/contributors/documentation/copy-guide.md#five-pay-attention-to-capitalization). I wasn't sure whether the block name should be sentence capitalization or title capitalization, but it appears that historically title capitalization has been used.

## How?

I have applied the two rules listed at the beginning to the block title. We have corrected the failed tests accordingly.

## Testing Instructions

Insert/Search the block that this PR has made changes to on the block editor and confirm that it appears with the correct label.
